build/yaml/crd/vpc/crd.nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -15,9 +15,9 @@ spec:
   scope: Cluster
   versions:
   - additionalPrinterColumns:
-    - description: NSXProject the Namespace associated with
+    - description: NsxProject the Namespace associated with
       jsonPath: .spec.nsxProject
-      name: NSXProject
+      name: NsxProject
```

Comment:
https://github.com/vmware-tanzu/nsx-operator/blob/main/build/yaml/crd/vpc/crd.nsx.vmware.com_vpcnetworkconfigurations.yaml#L20
wenqi's patch is NSXProject not NsxProject

please rebase main latest and re run make generated, make manifests.